test/task.h

```diff
@@ -184,6 +184,7 @@ enum test_status {
 #  define inline __inline
 # endif
 
+# if defined(_MSC_VER) && _MSC_VER < 1900
```

Comment:
But `#ifndef inline` is not working for me in MSVC2015, it steps right into the block.